client/bin/start.js

```diff
@@ -128,7 +131,12 @@ async function startDevClient(clientOptions) {
 
   const client = spawn(clientCommand, clientArgs, {
     cwd: resolve(__dirname, ".."),
-    env: { ...process.env, CLIENT_PORT },
+    env: { 
+      ...process.env, 
+      CLIENT_PORT,
+      INSPECTOR_SSL_CERT_PATH: process.env.INSPECTOR_SSL_CERT_PATH,
+      INSPECTOR_SSL_KEY_PATH: process.env.INSPECTOR_SSL_KEY_PATH
```

Comment:
If you have just spread process.env above, does it make any sense to explicitly spread these two variables? Will they not be included as part of `...process.env`?